Source/Runtime/MeasurementLink Measurement Server/MeasurementLink Measurement Server.lvlib

```diff
@@ -10,6 +10,7 @@
 	<Item Name="Helpers" Type="Folder">
 		<Property Name="NI.LibItem.Scope" Type="Int">4</Property>
 		<Property Name="NI.SortType" Type="Int">0</Property>
+		<Item Name="Create Enum Type Specialization.vi" Type="VI" URL="../Helpers/Create Enum Type Specialization.vi"/>
```

Comment:
You're going to have to merge in my changes from this PR: https://github.com/ni/measurementlink-labview/pull/209

Luckily it is a merge of an .lvlib, which is text, so it shouldn't be too difficult.

Reply:
> I assume you will update at least one example to use an enum? I think it would be useful to show how to use this VI. I assume you would call it directly from Get Type Specializations.vi, right?

None of the existing examples use enum so we are not intending to add enums to those examples, however we have a separate work item to create a [standalone example specifically for enums](https://ni.visualstudio.com/DevCentral/_backlogs/backlog/InstrumentStudio%20Platform/Features/?workitem=2394663) and that will come in a separate PR.

For remaining comments, I made changes to resolve them as suggested. Let me know if there are any issues with the way I did so.

Reply:
The Python examples for nidmm, nifgen, niscope, and nivisa use strings in place of enums and should be updated to use real enums.

The LabVIEW examples don't cover these drivers, so I guess they are fine.